admin/homebrew.py

```diff
@@ -144,17 +131,39 @@ def main():
     If no command line argument is provided, use the standard release
     location for the indicated version.
     """
-    version = get_version()
-    if len(sys.argv) < 2:
+    logging.basicConfig(level=logging.INFO)
+
+    parser = argparse.ArgumentParser(description='Process some integers.')
```

Comment:
This could be a bit more descriptive, if I run `--help` at the moment I get
```
usage: make-homebrew-recipe [-h] [--flocker-version FLOCKER_VERSION]
                            [--sdist SDIST] [--output-file OUTPUT_FILE]

Process some integers.
```